README.md

```diff
@@ -40,6 +42,20 @@ cmake .. -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=install
 make -j install
 ```
 
+### Install
+To install the SDK -
+
+```bash
+make install
+```
+
+### Debian Package
+To generate a Debian package for the SDK -
+
+```bash
+make package
```

Comment:
It's not clear from this how to do a build for a different CUDA toolkit version?

Reply:
The packaging is independent of the toolkit version. the packaging just puts the build libraries/headers into a deb package.
To build for a different cuda version cmake would need to be run with -DCUDA_TOOLKIT_DIR (i.e. the cmake command for setting a custom toolkit folder)

Reply:
Could we clarify this in the Readme?

Reply:
Yep, clarified that it only packages what is built by the `make` command and doesn't automatically support cross platform/CUDA support